corehq/apps/accounting/static/accounting/js/pricing_table.js

```diff
@@ -1,5 +1,5 @@
 hqDefine('accounting/js/pricing_table', function () {
-    var pricingTableModel = function (editions, current_edition, isRenewal) {
+    var pricingTableModel = function (editions, current_edition, isRenewal, isAnnualPricing) {
```

Comment:
This would be a good place for an options object instead of a parameter list (same comment as in the 30-day minimum PR).